lib/sql.js

```diff
@@ -1185,8 +1185,15 @@ SQLConnector.prototype.buildOrderBy = function(model, order) {
     return '';
   }
   var self = this;
-  if (typeof order === 'string') {
-    order = [order];
+  var supportedConnectors = ['mysql', 'postgresql', 'oracle', 'mssql'];
```

Comment:
I don't think we need to have an array of supported connectors here in the base class. If a connector doesn't implement `buildOrderByRandom()`, the default impl will throw.